apps/storefront/src/pages/QuoteDetail/index.tsx

```diff
@@ -358,10 +358,10 @@ function QuoteDetail() {
         snackbar.error(error.message);
       });
 
-      return false;
+      return true;
     }
 
-    return true;
+    return false;
```

Comment:
hey @icatalina, i need to change this because in this line https://github.com/bigcommerce/b2b-buyer-portal/pull/616/files#diff-518d8baa6ceb5f14406981623bf714117821b4adf07267757822901cd30e8a64R28 if i return true here, it will stop the process to checkout. But also it doesn't have much sense returning true when we have errors, should I rename this function? Any recommendation here?

Reply:
I made a change to the naming of these "proceedToCheckout..." function to `hasQuoteValidationErrorsXFlow` since it seems to make more sense with the return value we use to stop the checkout flow, let me know what you think.